openbas-model/src/main/java/io/openbas/database/model/Filters.java

```diff
@@ -36,6 +36,13 @@ public static class FilterGroup {
     @NotNull private FilterMode mode; // Between filters
     private List<Filter> filters = new ArrayList<>();
 
+    public static FilterGroup defaultFilterGroup() {
+      FilterGroup filterGroup = new FilterGroup();
+      filterGroup.setMode(FilterMode.or);
+      filterGroup.setFilters(List.of());
```

Comment:
List.of here might be problematic as it creates an immutable list. Someone could want to create a default filter group and then add filters to it. This would wause a runtime error in this case. Maybe new ArrayList<>() instead ?